styles/package.json

```diff
@@ -25,16 +25,19 @@
   ],
   "scripts": {
     "generate-layers": "tsx src/generate_layers.ts",
-    "dist-light": "mkdir -p dist/layers && npm run --silent generate-layers protomaps light > dist/layers/light.json",
-    "dist-dark": "mkdir -p dist/layers && npm run --silent generate-layers protomaps dark > dist/layers/dark.json",
-    "dist-white": "mkdir -p dist/layers && npm run --silent generate-layers protomaps white > dist/layers/white.json",
-    "dist-grayscale": "mkdir -p dist/layers && npm run --silent generate-layers protomaps grayscale > dist/layers/grayscale.json",
-    "dist-black": "mkdir -p dist/layers && npm run --silent generate-layers protomaps black > dist/layers/black.json",
+    "generate-styles": "tsx src/generate_styles.ts",
+    "dist-light": "mkdir -p dist/layers && npm run --silent generate-layers protomaps light en Latin > dist/layers/light.json",
```

Comment:
let's remove `dist/layers` in favor of `dist/styles`. I think add a placeholder URL like `https://example.com/tiles.json` is good to make it obvious the `sources.protomaps.url` must be modified.